dftparse/wien2k/absorp_parser.py

```diff
@@ -0,0 +1,47 @@
+from ..core import BlockParser
+
+
+def _parse_absorption(line, lines):
+    """Parse Energy, Re sigma xx, Re sigma zz, absorp xx, absorp zz"""
+    # skip the first two lines after the rule line
+    next(lines)
+    newline = next(lines)
+
+    energies = []
+    wavelengths = []
+    frequencies = []
+    re_sigma_xx = []
+    re_sigma_zz = []
+    absorp_xx = []
+    absorp_zz = []
+
+    while newline[0] != "#" and len(newline.split()) == 5:
+        split_line = newline.split()
+
+        energies.append(float(split_line[0]))
+        wavelengths.append(float(split_line[0])/1240)
```

Comment:
Why divide by 1240?

Reply:
to convert energy to wavelength

Reply:
That type of conversion shouldn't be in `dftparse`; you could do it in `pif-dft`.  `dftparse` is just for extracting structured data from the text files.

Reply:
ahh okay, i'll remove these and the attached tests